ml-proto/test/memory.wast

```diff
@@ -95,7 +95,7 @@
     (loop
       (if
         (i32.eq (get_local 0) (i32.const 0))
-        (br 1)
+        (br 2)
```

Comment:
is it the if that has an implicit block, or the then and else branches that do? because I see they can have a label name on them now? must it be the same for both of them if there is?